src/main.css

```diff
@@ -59,3 +59,13 @@ body > main {
 .katex-display .katex-html {
     overflow: auto hidden;
 }
+
+/* Custom scrollbar */
+::-webkit-scrollbar {
+    width: 0.3em !important;
```

Comment:
It seems like the `important` is unnecessary?

Reply:
I didn't add `!important` at first, but I found it in one of the themes I used to use.  So I added it, maybe it wasn't necessary.

Reply:
Okay, it should be fixed.
Now let us make the width wider, to make the a11y better then. I think the `.5em` or so is a value for good.